code/modules/admin/verbs/playsound.dm

```diff
@@ -192,9 +208,10 @@
 			to_chat(src, span_boldwarning("Non-http(s) URIs are not allowed."), confidential = TRUE)
 			to_chat(src, span_warning("For youtube-dl shortcuts like ytsearch: please use the appropriate full URL from the website."), confidential = TRUE)
 			return
-		web_sound(usr, web_sound_input)
+		var/number_input = tgui_input_number(usr, "What range would you like to play it in? (leave empty for everyone)", "Play Internet Sound", null)
```

Comment:
im starting to not like how much of tgui yes/no clickthroughs we're starting to have here just to play a sound

Reply:
Yeah that's fair but I was too lazy to refactor it and move the code

Reply:
By that I mean if I were to make a new proc for it

Reply:
ideally this would be its own dedicated settings menu with all the options in one area

Reply:
Fikou pinged me on discord with it, might do that a bit later